README.md

```diff
@@ -33,11 +33,19 @@ Developers on a OS for which there is no binary package, or who would like to st
 1. Install the Xcode command-line tools.
 1. Download version 1.52 of [Boost](https://sourceforge.net/projects/boost/files/boost/1.52.0/).
 1. Set the `BOOSTDIR` environment variable to the location containing this boost installation.
-1. Install [Mono](http://www.mono-project.com/download/stable/).
-1. Install a [JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html). FoundationDB currently builds with Java 8.
+1. Install [Mono] from ports(https://www.freshports.org/lang/mono).
+1. Install a [JDK] from ports(http://www.oracle.com/technetwork/java/javase/downloads/index.html). FoundationDB currently builds with Java 8.
 1. Navigate to the directory where you checked out the foundationdb repo.
 1. Run `make`.
 
+#### FreeBSD
+
+1. Check out this repo on your server.
+1. Install dependencies from ports using the following command: pkg install mono boost-libs gmake bash
+1. (Optional) Install a [JDK](https://www.freshports.org/java/openjdk8/) for Java Bindings. FoundationDB currently builds with Java 8.
```

Comment:
I did `# cd /usr/local/ports/java/openjdk8/ ; make install`, and afterwards when trying to `gmake bindings` still got

`#include <jni.h>` --> `fatal error: 'jni.h' file not found`

Which is probably me doing something wrong, but I didn't have a chance to dig into this.

Reply:
Path to jni.h got resolved and similar to Linux except OpenJDK8 settings are being used now. This opened up a lot more Java questions for me and the new section needs a lot of cleanup but not hitting any error messages there anymore.